src/types/BaseField.ts

```diff
@@ -49,14 +51,44 @@ const writableBaseFieldSchema: JSONSchemaType<WritableBaseField> = {
 			enum: Object.values(BaseFieldScope),
 		},
 	},
-	required: ['label', 'description', 'shortCode', 'dataType', 'scope'],
+	required: ['shortCode', 'dataType', 'scope'],
 };
 
+const writableBaseFieldWithLocalizationsSchema: JSONSchemaType<WritableBaseFieldWithLocalizations> =
+	{
+		type: 'object',
+		properties: {
+			shortCode: {
+				type: 'string',
+			},
+			dataType: {
+				type: 'string',
+				enum: Object.values(BaseFieldDataType),
+			},
+			scope: {
+				type: 'string',
+				enum: Object.values(BaseFieldScope),
+			},
+			localizations: {
+				type: 'array',
+				items: writableBaseFieldLocalizationWithBaseFieldContextSchema,
+			},
+		},
+		required: ['shortCode', 'dataType', 'scope', 'localizations'],
+	};
```

Comment:
Likewise, as a new schema, we should disallow unrecognized keys here, too:

```suggestion
        additionalProperties: false,
	};
```